src/frontend/public/index.html

```diff
@@ -10,7 +10,6 @@
       content="Web site created using create-react-app"
```

Comment:
Is this meta relevant to our project i.e. the description etc? Might be useful to check the content of this file against other NHS Digital react projects